code/starfield/starfield.cpp

```diff
@@ -869,19 +870,21 @@ void stars_post_level_init()
 	for (i=0; i<MAX_STARS; i++) {
 		dist = dist_max;
 		while (dist >= dist_max) {
-			v.xyz.x = (float) ((myrand() & RND_MAX_MASK) - HALF_RND_MAX);
-			v.xyz.y = (float) ((myrand() & RND_MAX_MASK) - HALF_RND_MAX);
-			v.xyz.z = (float) ((myrand() & RND_MAX_MASK) - HALF_RND_MAX);
+			v.xyz.x = (float) ((util::Random::next() & RND_MAX_MASK) - HALF_RND_MAX);
+			v.xyz.y = (float) ((util::Random::next() & RND_MAX_MASK) - HALF_RND_MAX);
+			v.xyz.z = (float) ((util::Random::next() & RND_MAX_MASK) - HALF_RND_MAX);
 
 			dist = v.xyz.x * v.xyz.x + v.xyz.y * v.xyz.y + v.xyz.z * v.xyz.z;
 		}
 		vm_vec_copy_normalize(&Stars[i].pos, &v);
 
 		{
-			red= (ubyte)(myrand() % 63 +192);		//192-255
-			green= (ubyte)(myrand() % 63 +192);		//192-255
-			blue= (ubyte)(myrand() % 63 +192);		//192-255
-			alpha = (ubyte)(myrand () % 192 + 24);	//24-216
+			// DISCUSSME: the max value for these ranges is 254 and not 255. Do we fix it?
+			red= (ubyte)(util::Random::next(63) +192);		//192-255
+			green= (ubyte)(util::Random::next(63) +192);		//192-255
+			blue= (ubyte)(util::Random::next(63) +192);		//192-255
+			// DISCUSSME: despite the comment, the range is actually 24-215. Do we fix it?
+			alpha = (ubyte)(util::Random::next(192) + 24);	//24-216
```

Comment:
Despite my general position that we should preserve existing behaviour over apparent intention, this is a) a tiny change in purely (?) cosmetic code, which is b) mostly obsolete in FSO today. Of course, given the latter, you wonder if it's worth changing it at all. I don't see why you don't just refactor these into two-argument range calls to `next()`, though.

Reply:
I'm on the side of matching the code to the comments here.